src/main/java/fr/lesprojetscagnottes/core/slack/controller/SlackTeamController.java

```diff
@@ -69,22 +70,67 @@
     public SlackTeamModel getById(Principal principal, @PathVariable("id") Long id) {
 
         // Verify that ID is correct
-        if(id <= 0) {
+        if (id <= 0) {
             LOGGER.error("Impossible to get Slack team by ID : ID is incorrect");
             throw new BadRequestException();
         }
 
         // Verify that entity exists
         SlackTeamEntity entity = slackTeamRepository.findById(id).orElse(null);
-        if(entity == null) {
+        if (entity == null) {
+            LOGGER.error("Impossible to get Slack team by ID : Slack team not found");
```

Comment:
The `LOGGER` class attribute can be replaced with `log` provided by the `@Slf4j` annotation

Reply:
considered